docs/content/toc.textile

```diff
@@ -16,7 +16,7 @@ h2. Data Ingestion
 * "Schema Design":../ingestion/schema-design.html
 * "Schema Changes":../ingestion/schema-changes.html
 * "Realtime Ingestion":../ingestion/realtime-ingestion.html
-* "Batch Ingestion":../ingestion/batch-ingestion.html
+* "Batch ingestion or Re-indexing or Delta ingestion":../ingestion/batch-ingestion.html
```

Comment:
i think we should add batch-delta-ingestion.md and link it from the TOC. The document can be short but we can move the contents around reingestion out of batch-ingestion or repeat the relevant parts